test/threeid-provider.test.js

```diff
@@ -65,7 +70,7 @@ describe('ThreeIdProvider', () => {
   beforeAll(async () => {
     tmpFolder = await tmp.dir({ unsafeCleanup: true })
     ipfs = await Ipfs.create(genIpfsConf(tmpFolder.path))
-    ceramic = await Ceramic.create(ipfs, { stateStorePath: tmpFolder.path + '/ceramic/' })
+    ceramic = await Ceramic.create(ipfs, { stateStoreDirectory: tmpFolder.path + '/ceramic/', anchorOnRequest: false })
```

Comment:
Not needed, but we could just use the `jest-environment-ceramic` instead of manually setting up a node here.

Reply:
See https://github.com/ceramicstudio/js-idx/blob/master/packages/tools/test/lib.test.ts#L1-L3

Reply:
oh nice, did know we had that, but yeah will add as part of future cleanup story